source/examples/xgboost-randomforest-gpu-hpo-dask/notebook.ipynb

```diff
@@ -0,0 +1,752 @@
+{
```

Comment:
<p>The airline dataset url is broken - do we know of another source? If not we probably should change this to something like:</p><p> </p><blockquote>For the demo, we will be using an Airline dataset that has about 116 million entries with 13 attributes that are used to determine the delay for a given airline. The aim of this problem is to predict the arrival delay. To test this, we have modified this problem o serve as a binary classification problem to determine if the airline will be delayed (True) or not (False).</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/deployment/pull/174/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
I manually downloaded the same dataset from S3 bucket and uploaded to Azure blob storage ... but yes i can also add that description (if unable to link the storage url).

Thanks for catching this, i forgot to update it.

Reply:
I have updated it with the new public [S3 bucket](https://s3.console.aws.amazon.com/s3/buckets/rapidsai-cloud-ml-sample-data?region=us-west-2&tab=objects)